extensions/wikia/VideoHandlers/VideoHandlers.setup.php

```diff
@@ -123,6 +123,9 @@
 	$wgHooks['ParserFirstCallInit'][] = 'VideoHandlerHooks::initParserHook';
 }
 
+$wgHooks['VideoInfoSaveToCache'][] = 'VideoHandlerHooks::onClearVideoCache';
+$wgHooks['VideoInfoInvalidateCache'][] = 'VideoHandlerHooks::onClearVideoCache';
```

Comment:
`onClearVideoCache` looks strange here. I was under impression that we're using `on` prefix when we related to hook name. Maybe we could use `clearVideoCache` here?